src/box/box.cc

```diff
@@ -1946,12 +1958,13 @@ box_listen(void)
 {
 	if (box_check_listen() != 0)
 		return -1;
-	int count = cfg_getarr_size("listen");
-	const char **uris = (const char **)xmalloc(count * sizeof(char *));
-	for (int i = 0; i < count; i++)
-		uris[i] = cfg_getarr_elem("listen", i);
-	int rc = iproto_listen(uris, count);
-	free(uris);
+	struct uri_set uri_set;
+	cfg_get("listen");
```

Comment:
Maybe, add a `cfg_*` helper that would hide all `tarantool_L` manipulations?

Reply:
Yes, good idea!